### PR TITLE
Better placement of Login/Register button

### PR DESCRIPTION
**Pull request by adarshguptacse18**

### CODE REVIEW

File: src/components/Intro.vue [hunk 1]
```diff
@@ -47,6 +58,9 @@ export default {
     player() {
       return this.$refs.youtube.player;
     },
+    ismobile(){
+      return this.$mq === "xs" || this.$mq === "sm";
```
Comment (shivanshs9) [inline src/components/Intro.vue:62]: There is actually a function, **isMinimal** in js/utils.js
You can use that here.

File: src/components/Intro.vue [hunk 2]
```diff
@@ -19,10 +19,21 @@
         provides the perfect platform for fresh enthusiasts, as well as the
         experienced ones, to code together and compete for ultimate glory.
       </p>
+      <div v-if="!ismobile" :class="$style.buttonContainer">
+        <div :class="$style.link">
+          <router-link to="/login"
+            :class="[$style.linkText]"
+          >
+            <h4>Register / Login</h4>
```
Comment (shivanshs9) [inline src/components/Intro.vue:27]: I think it's a good idea to show "Dashboard" here and redirect to **/dashboard** if the user is logged in.
To check logged in, use **this.$store.getters.isLoggedIn** getter in a computed property

Comment (adarshguptacse18) [reply src/components/Intro.vue]: Done

File: src/components/Intro.vue (hunk 1 above)
Comment (adarshguptacse18) [reply src/components/Intro.vue]: Done

**Pull request merged**